src/WebSdk/Publish/Targets/Microsoft.NET.Sdk.Publish.targets

```diff
@@ -93,10 +93,6 @@ Copyright (C) Microsoft Corporation. All rights reserved.
     <PublishIntermediateOutputPath Condition="'$(DockerPublish)' == 'true'">obj/Docker/publish/</PublishIntermediateOutputPath>
 
     <EFSQLScriptsFolderName Condition="$(EFSQLScriptsFolderName) == ''">EFSQLScripts</EFSQLScriptsFolderName>
-
-    <!-- If TrimMode has not already been set, default to partial trimming, as AspNet is not currently fully trim-compatible -->
-    <!-- We allow the full trimming default with PublishAot enabled, because the parts that are not trim-compatible are not aot-compatible either -->
-    <TrimMode Condition="'$(TrimMode)' == '' and '$(PublishTrimmed)' == 'true' and '$(PublishAot)' != 'true'">partial</TrimMode>
```

Comment:
Uggh, I just realized that we probably should be leaving `TrimMode=partial` for TFMs below `net8.0`. I don't think we should be changing behavior for people when they upgrade their SDK. Only when they retarget from `net6/7.0` => `net8.0`.